---
isObsidianImport: true
created: Thu May 26 2022 18:40:25 GMT-0600 (Mountain Daylight Saving Time)
lastModified: Mon Jun 20 2022 09:28:18 GMT-0600 (Mountain Daylight Saving Time)
---
# BIOCH 200 | INTRODUCTORY BIOCHEMISTRY
**Spring 2022** #Class 

---

## Main Block
### Time
| Block    | Days  | Time |
| -------- | ----- | ---- |
| Lectures | MTWRF | 10:30 - 11:40 am     |

### Rubric
| Activity | Weight |
| -------- | ------ |
| Quiz 1   | 1%     |
| Midterm  | 35%    |
| Quiz 2   | 14%    |
| Final    | 50%    | 

### Outline

### Due Dates

---

## [[Biomolecules]]
The foundation of this course, they are [[Monomer]]s which form [[Polymer]]s

The primary atoms are
Hydrogen Carbon Nitrogen Oxygen Phosphorus Sulfur
**[[PHONCS]]**

Na Mg K Ca Cl are the most important metals that these [[Biomolecules]] will interact with for this course; however, Fe is the primary focus of this course.

The colours for each atom in [[Biomolecules]] are:
Oxygen = Red
Carbon = Grey/Black
Hydrogen = White
Nitrogen = Blue
Sulfur = Yellow
Phosphorus = Orange

At around PH ~7 the [[Carboxylic Acid]] [[Functional Group]] typically turns into a [[Carboxylate]] [[Functional Group]].

The difference between the [[Ester]] and [[Ether]] is that Esters consist of a [[Carbonyl]].

[[Ester]] [[Ether]] and [[Amide]] groups can be [[Functional Group]] but also a [[Biochemical Linkage]].

[[Carbonyl]] AND [[Carboxylic Acid]] ARE DIFFERENT!!!

HCSNO
least electroneg to most electroneg

There are 2 main types of interactions: [[Electrostatic Force]]s and [[Covalent Bond]]s, as well as the [[Hydrophobic Effect]], which unlike the other 2, is **not** a bond.

Is a molecule is more hydrophobic, the term [[Amphipathic]] is used.
If a molecule is more hydrophilic, the term [[Amphiphillic]] is used.
They both mean the same thing though.
Philli -> Philli

Both types of molecules typically form [[Micelle]]s or [[Leaflet]]s when in water.

## [[Nucleotide]] and [[Nucleic Acid]]s
The main difference between [[Nucleic Acid]]s and [[Protein]]s is:
[[Nucleic Acid]]s have high Phosphorus and NO Sulfur
[[Protein]]s have high Sulfur and NO Phosphorus

[[Nucleic Acid]]s are made from joining [[Nucleotide]]s
[[Nucleotide]]s are made from joining a [[Nucleoside]] and a [[Phosphate]]
[[Nucleoside]] are made from a [[Nucleoside Base]] and a [[Sugar]]

Lone pairs in resonance for Nitrogen cannot accept [[Hydrogen Bond]]s.
Nitrogen can only **accept** if it is **double AND single bonded**, as well as **NOT** bonded to a carbon **with** oxygen.

Caffeine is a form of Purine.

The differences between [[Deoxyribonucleic Acid]] and [[Ribonucleic Acid]] is that RNA has an extra O meaning more [[Hydrogen Bond]]s, and RNA uses Uracil while DNA uses Thymine.

Uracil and Thymine are used in [[Ribonucleic Acid]] and [[Deoxyribonucleic Acid]] respectively, the rest are used in both.

| [[Nucleoside Base]] | In [[Deoxyribonucleic Acid]] | In [[Ribonucleic Acid]] | Connects to      |
| ------------------- | ---------------------------- | ----------------------- | ---------------- |
| Uracil              | No                           | Yes                     | Adenine          |
| Thymine             | Yes                          | No                      | Adenine          |
| Adenine             | Yes                          | Yes                     | Uracil / Thymine |
| Guanine             | Yes                          | Yes                     | Cytosine         |
| Cytosine            | Yes                          | Yes                     | Guanine          |

[[Phosphate]]s can be linked to different carbons in a [[Sugar]] but it would need to be specified if it is not linked to 5'.

3' - 5' [[Phosphodiester Bond]]s are used to form [[Nucleic Acid]]s from [[Nucleotide]]s

They are written from left to right starting from the 5' carbon to 3' carbon

To break [[Phosphodiester Bond]], [[Enzymatic Hydrolysis]] can occur. If it separates as the 5' side, the resulting molecules are unusual with uneven phosphorylation, so this will typically occur at the 3' side.

With [[Alkaline Hydrolysis]], quarter of the phosphates attach to 2', and quarter of the phosphates attach to the 3'. The rest half of these are attached with a [[Phosphate]] only at the 5'.

If there are no [[Hydroxyl]]s in the sugar, then the [[Nucleoside]] is called **dideoxy**

Single stranded [[Deoxyribonucleic Acid]] absorbs more UV than double stranded [[Deoxyribonucleic Acid]] does.

[[Protein]] absorbs around ~280nm UV
[[Deoxyribonucleic Acid]] absorbs around ~260nm UV

If there is [[Protein]] contamination, then the 260/280 ratio will decrease.

Pure DNA has a ratio of about 1.8-1.9

If [[Hydrogen Bond]]s are to occur between [[Functional Group]]s, then they should not be near Water, or else it will bond with water.

[[Hydrogen Bond]]s between bases will occur using 1 in Purine and 3 in Pyrimidine.

DNA is anti-parallel, so one is going normally from 5' to 3', while the other is going from 3' to 5'.

The secondary structure of DNA is supported by [[Base Stacking Interaction]]s ([[Van der Waals]] and [[Hydrophobic Effect]]) typically.

The main form of DNA is B-form and is a right hand twist.

Major grooves are the larger gaps
Minor grooves are the smaller gaps

They are also hydrophobic inside, but very polar on the outside.

Both strands **must not** be identical.

When the complementary is asked, the strand must be reversed as it starts from 3'

[[Nucleic Acid]]s have a sense of direction because the ends of each are geometrically distinct.

DNA Melting is to change double stranded DNA to single stranded, also known as [[Denaturation]]

In [[Renaturation]] or DNA Hybridizing the single stranded DNAs are combined back to double stranded DNA. There are 2 processes:
- Nucleation (slow depending on DNA complexity to ensure proper base pairing)
- Zippering (fast)

The thermal stability of the DNA is dependant on 
- strand length (smaller length = lower melting temperature)
- composition + sequence (AT > GC = lower melting temperature)
- salt (lower salt = lower melting temperature)
- pH (pH > 10 = lower melting temperature)

**WHEN PH=PKA THEN HA AND A- WILL BOTH EQUALLY EXIST IN CONCENTRATION**

AT-only DNA has very low melting temperature (~70C)
GC-only DNA has very high melting temperature (~110C)

GC has higher [[Base Stacking Interaction]]s

While DNA is only single/double stranded, RNA is mostly single stranded but can have complex shapes.

[[Intrastrand Base Pairing]] is different from [[Interstrand Base Pairing]] where the first is based on base pairing in the **same strand**, while the second is based on base pairing in **different strands**.

DNA is far more stable than RNA, DNA does not degrade

## Protein Structure
A [[Protein]]'s function is determined by its structure

**WHEN PH=PKA THEN HA AND A- WILL BOTH EQUALLY EXIST IN CONCENTRATION**

| [[Functional Group]] | pKa    |
| -------------------- | ------ |
| [[Carboxylic Acid]]  | 2      |
| [[Amino]]            | 9.5    |
| [[Thiol]]            | 9      |
| [[Hydroxyl]]         | Varies | 

The charge of [[Amino Acid]]s can change at varying degrees of pH.

As pH increases, [[Amino Acid]]s become more negative. Decreasing pH increases positivity.

Most [[Amino Acid]]s are typically [[Zwitterions]] and all of them are [[Chiral Molecule]]s as well

The [[Amino Acid]]s can be grouped based on their R group:
- Hydrophobic (nonpolar)
- Polar (uncharged)
- Charged (most polar [[Functional Group]])

However hydrophobic [[Amino Acid]]s can still be soluble and [[Amphipathic]].

Aromatic/[[Phenyl]] rings absorb at 280 nm

2 or more Cysteine bonds can form [[Disulphide Bond]]s in an oxidizing environment.
This is called Cystine.

Asp usually has 1 Carbon, Glu usually has 2 carbons

[[Amino Acid]]s are typically connected with [[Peptide]] Bonds (Nitrogen from 1 [[Amino]] to Oxygen from another [[Carboxylate]]) which forms an [[Amide]].

The $\alpha$-Carbon is the carbon connected to the nitrogen on the other side of the [[Peptide]] bond (the one not connected to a [[Carbonyl]])

The side that connects to the other's nitrogen is N-terminus, and is + at pH 1 and pH 7
The side that connects to the other's [[Carbonyl]] is C-terminus, and is - at pH 7 and pH 14 

2 [[Amino Acid]]s joined is called a dipeptide. 3 = tri, e.t.c.

Peptides/oligopeptides are typically synthetic [[Peptide]]s
[[Polypeptide]]s are long chain [[Amino Acid]] [[Peptide]]s
[[Protein]]s are [[Peptide]]s with biological function

[[Peptide]]s have a sense of direction, so the order of joining matters. Groups in [[Peptide]]s have no pKa.

[[Peptide]] bonds consist of resonance, so the bond formed contains a partial bond as well. These bonds absorb UV at ~230 nm 

[[Peptide]]s are planar and rigid. The backbone consists of the main carbons and the [[Peptide]] bonds formed.

The secondary structures of [[Protein]]s consist of the form of shape the [[Peptide]] backbones take.

Loops are typically irregular structures, while [[sheet]]s or [[Helix]]es are normal. Helixes are also right hand twists like DNA.

The [[Carbonyl]] oxygen combines with the [[Amine]] 4 residues later, so C1 -> N5.

Glycine and Proline are typically not found in the middle of [[Protein]] [[Helix]]es.

Only terminal groups in [[Peptide]]s retain charge.

Sheets are typically formed by [[Peptide]]s that are side-by-side. They can be parallel or anti-parallel. Anti-parallel [[Peptide]]s are more straight and slightly more stable.

Loops connect the sheets. Anti-parallel loops are much shorter than Parallel loops. Loops can also be of different shapes as well.

**A SHEET DOES NOT INCLUDE THE LOOPS**

Helixes are stabilized by [[Hydrogen Bond]]s within the backbone of the same strand.
Sheets are stabilized by [[Hydrogen Bond]]s between strands.

Irregular structures like loops does not mean disordered, just not flexible/generic in shape.

[[Prosthetic Group]]s are non-protein structures that are **tightly bounded** to proteins, such as [[Heme]] and FAD. 

There are 2 types of tertiary structures:
- Fibruous Proteins (long + insoluble in water)
- Globular Proteins (compact + soluble in water)

Hydrophobic portions of a [[Protein]] typically are more within the structure instead of outside, but they can be outside.

Secondary structures can also be [[Amphipathic]], with more internal structures being more hydrophobic.

Secondary and Tertiary structures are fine-tuned by [[Hydrogen Bond]]s, which can also form between the backbone and sidechain R groups.

Ion pairs help [[Hydrogen Bond]] in fine-tuning secondary and tertiary structures between (positive and negative) charged [[Amino Acid]]s

[[Disulphide Bond]]s/Bridges stabilize crosslinks for extracellular [[Protein]]s

A [[Domain]] is a segment of a [[Polypeptide]] which is folded into its own unit with its own core.

A [[Motif]] is a very short segment of a [[Polypeptide]] which has its own 3D shape.

[[Prosthetic Group]]s can provide help in creating structure or reactivity.

[[Coordination Bond]]s are [[Covalent Bond]]s regularly seen when [[Prosthetic Group]]s are in play.

Globular [[Protein]]s are not as stable as Fibruous [[Protein]]s, they are stabilized by noncovalent forces.

## Protein Function

The function of a [[Protein]] is 100% decided by structure
For [[Protein]]s, a monomer is when it has 1 [[Polypeptide]] chain, not the same as [[Monomer]]s for [[Nucleotide]]s. More than 1 [[Polypeptide]] chain means that it is an [[Oligomer]]

[[Hemoglobin]] binds $O_2$ in the lungs and releases it in the tissues
[[Myoglobin]] helps reserve $O_2$ in situations such as intensive exercise and helps diffuse $O_2$ in muscle and improve tissue health.

[[Ligand]]s dictate the function of a [[Protein]] based on which ligands a protein can bind to.

The dissociation constant of a [[Ligand]] for a [[Protein]] is the concentration of the [[Ligand]] when 50% saturation occurs. Higher $K_d$ means more concentration needed, and less affinity of the [[Ligand]] to the protein.

[[Myoglobin]] typically has 1 domain and no sheets. To store $O_2$ it contains a [[Prosthetic Group]] called [[Heme]].

[[Heme]] is connected to a Histidine and it is named **His F8**. The oxygen connected to [[Heme]] can also connect to another **distal** Histidine which is notated **His E7**. His E7 helps with avoiding binding with $CO$ by forcing connections with [[Heme]] and His E7.

When muscle rests, there is high $O_2$ concentration, but loses most of it when the muscles are active.

When a [[Prosthetic Group]] is added to a [[Protein]], it is a **holoprotein**.
When a [[Prosthetic Group]] is not added to a [[Protein]], it is an **apoprotein**.

While [[Myoglobin]] does not have quaternary structure, [[Hemoglobin]] does.

[[Hemoglobin]] can bond 4 $O_2$ compared to 1 by [[Myoglobin]], and contains 4 chains.

Conservative substitutions of [[Amino Acid]]s can possibly preserve most structure and function of a [[Protein]]. Typically substitutions of similar classification.

[[Hemoglobin]] subunits bind to $O_2$ in the same manner as [[Myoglobin]].

Some relationships of [[Ligand]]s and [[Protein]]s are sigmoidal, where they have more affinity to a [[Protein]] when more [[Ligand]]s are joined.

[[Hemoglobin]] has a sigmoidal relationship with [[Ligand]]s.

[[Hemoglobin]] in the blood in lungs will have very high $O_2$ concentration. Tissues on the other hand will have very low $O_2$ in the [[Hemoglobin]] in its blood.

[[Hemoglobin]] and [[Myoglobin]] have both similar functions in binding and releasing $O_2$. [[Myoglobin]] however transports and stores $O_2$ within tissues, while [[Hemoglobin]] transports $O_2$ from lungs to tissues.

[[Hemoglobin]] can change its affinity with $O_2$ by changing its structure:
- When it is tense, it is lower in affinity (DeoxyHB)
	- More salt bridges formed
	- Larger core gap
- When it is relaxed, it is higher in affinity (OxyHB)
	- Fewer salt bridges formed
	- Smaller core gap

[[Effectors]] observe [[Allostery]], and there are 2 types: 
- Homoallostery (same compound binding is affected)
	- [[Hemoglobin]] with $O_2$
- Heteroallostery (different compound binding is affected)
	- Activators increase binding affinity of certain [[Ligand]]s
	- Inhibitors decrease binding affinity of certain [[Ligand]]s

If [[Protein]]s don't have quaternary structure, they likely do not experience [[Allostery]].

When [[Hemoglobin]] raises affinity with $O_2$ from T to R state, the iron typically moves up into the [[Heme]] plane and is now easier for binding.

While $O_2$ favours R state, another chemical BPG, as well as H+ itself encourage T state.

BPG binds into the centre of a DeoxyHB, and interact with His which causes discouragement of $O_2$ binding

$CO_2$ also favours the T state as well by binding to N-terminal [[Amino Acid]]s, and is formed with high cellular activity.

H+ facilitates/enhances BPG binding, which in turn favours the T state.

Higher pH causes higher affinity for $O_2$ and favouring of the R state due to lower H+, and this is the case for lungs.

Vice-versa for tissues.

Sickle Cell Amenia occurs due to an exposure of an area within the structure of the [[Hemoglobin]], allowing Valine to bind, which is hydrophobic.

Fetal [[Hemoglobin]] also has a higher $O_2$ affinity than adult [[Hemoglobin]] does.

Histidine has 3 specific roles depending on the residues for [[Hemoglobin]]:
- His F8
	- [[Heme]] Attachment
	- Proximal His
- His E7
	- Assist in binding $O_2$
	- Decreases $CO$ affinity
	- Helps with the binding site's geometry
	- Distal His
- 4 His in central cavities
	- BPG binding

## [[Enzyme]]s
[[Enzyme]]s allow better efficiency and multistep processes for using energy, and allows faster digestion.

Typically, heat, and catalysis are the 2 main methods of speeding a reaction. Heat is not easily controllable, so [[Enzyme]]s are used very often by bodies.

A few [[Ribonucleic Acid]]s can be [[Enzyme]]s but they are typically [[Protein]]s.

[[Ligand]]s are typically not used as terminology for reactions [[Enzyme]]s are taking part in.

[[Phosphoanhydride Bond]]s are high energy bonds that break -30 kJ which is far more than -25 kJ

All high energy bonds are typically very easily broken down into more stable molecules, and increase more entropy.

[[Gibbs Free Energy]] will tell us if the reaction will occur or not, but it does not tell us the [[Reaction Rate]] of the reaction. A larger amount of [[Gibbs Free Energy]] does not mean a faster rate; HOWEVER, it tells how spontaneous/likely a reaction is.

The [[Reaction Rate]] is determined by the [[Transition State]] of a reaction.

The [[Activation Energy]] is lowered by [[Enzyme]]s by:
- Removing reactants from aqueous environments
- Effecting the orientation and distance of molecules to more favourable ones
- Add transitional reaction steps to the main reaction
- Stabilizing [[Transition State]]s

[[Substrate Binding Region]]s typically change in shape as the reactions between reactants occurs, but typically returns back.

Desolvation/removing reactants from aqeous environments has many advantages:
- Removes water interference in reactions
- Far more easy to form H-Bonds
- Reduces the [[Activation Energy]] caused by water.

[[Enzyme]]s also increase the rate by taking part in the reaction themselves via different forms of catalysis:
- [[Acid-Base Catalysis]]
- [[Nucleophilic Catalysis]]

The active site on an [[Amino Acid]] binds the [[Transition State]] better than the reactants. The lock and key model is not as effective as the induced fit model as it encourages the shape of the [[Transition State]] for reactants. This is considered as **Preferential transition state stabilization**

The more tightly the [[Enzyme]] binds the [[Transition State]] the greater the effect

Lower $K_d$ for [[Ligand]] binding means higher affinity. At the plateau of the rate curve, all active sites are saturated

$K_m$ is typically used to calculate the affinity of the [[Enzyme]] as a substrate, based on plateau velocity / 2.

Many short term mechanisms occur to regulate [[Enzyme]] activity:
 - [[Competitive Inhibition]] can reduce the saturation levels of a given [[Ligand]]
 - [[Allostery]] can increase or decrease the catalytic effect between [[Ligand]]s from cases like Ionic Signals
 - Reversible covalent modification 

Some are long term as well:
- Gene expression regulation, affecting the number of enzyme molecules 
- Changes in Subcellular localization, changing the number of enzyme molecules which are favorable for the desired catalysis

[[Competitive Inhibition]] decreases the affinity (increases the apparent $K_m$) but doesnt change the max velocity

Inhibitors that have [[Transition State]] shapes instead of reactant shape are more effective as inhibitors.

[[Allostery]] causes a sigmoidal shape to the [[Reaction Rate]] graph, and is typically of a quaternary structure.

It is assumed [[Homoallostery]] unless it is stated there are other [[Ligand]]s

[[Allostery]] activators favour the R state and increases affinity
[[Allostery]] inhibitors favour the T state and reduces affinity

Protein [[Kinase]] causes Protein-OH + ATP -> ADP + Protein.
To revert the protein, water is added and insoluble phosphate is produced. This is completely reversible

## Membranes
3 main sections:
- [[Fatty Acid]]
- [[Triglycerides]] (Fat)
- [[Lipid]] (Cholestrol)

[[Fatty Acid]] are long hydrocarbons with a [[Carboxylic Acid]] attached.
If a [[Fatty Acid]] has no double bonds it is saturated
If a [[Fatty Acid]] has 1 double bond it is monounsaturated
If a [[Fatty Acid]] has 2+ double bonds it is polyunsaturated

These are usually [[Amphipathic]] and most double bonds are cis. If the double bonds are trans it must be stated. 
trans double bonds are in zig-zags
cis double bonds are straight

Carbons are counted from the [[Carboxylic Acid]]. The C attached to the acid is 1. 2 is alpha, 3 is beta, and the last one is omega.

When more and more double bonds are added into a [[Fatty Acid]] the effectiveness in packing becomes less due to the turning that occurs of the tail from the double bonds.

Longer [[Fatty Acid]]s melt at higher temperatures, and stack better.
More saturation melts at higher temperatures.

in trans double bonds the kinks and turning is far smaller.

When [[Fatty Acid]]s are put into water, [[Micelle]]s are formed.

[[Fatty Acid]]s typically is stored as a [[Lipid]] via [[Ester]] bonds to Glycerol. Mixed TAGs are most common (different [[Fatty Acid]]s joined).

When joined into a [[Lipid]] the counting starts from the omega Carbon.

Glycerophospholipids and Spingolipids are very similar in shape, but Sphingolipids have very big polar heads.

[[Cholestrol]] has a different shape from the 2, where it has a ring-like structure and a small polar head instead of 2 tails.

Glycerophospholipids and Spingolipids have only 2 [[Fatty Acid]] tails, Glycerophospholipids however do have a phosphate structure connected to the end of the 3rd tail.

[[Cholestrol]] is NEVER found in bacteria, and is weakly [[Amphipathic]], and have a [[Sterol]] to the polar end. They will have 27 carbons.

When [[Amphipathic]] molecules have 1 tail, they typically form [[Micelle]]s, but when they have 2 tails, they create [[Bilayer]]s, with each end's non-polar end called a Leaflet.

[[Bilayer]]s forming a circle similar to [[Micelle]]s are called [[Liposome]]s.

[[Cholestrol]] cannot create [[Bilayer]]s because it is not [[Amphipathic]] enough, and does not have 2 tails. It does not form membranes by itself at all.

[[Membrane]] are [[Lipid]]s that carry [[Protein]]s and [[Carbohydrate]]s

When heating a [[Bilayer]] the tails start moving far more rapidly and become more disordered, making them less gel-like.

[[Cholestrol]] keeps the [[Bilayer]] from being too fluid, and also prevents phospholipids from clustering together.

Transition/melting temperatures are far more sharp for artificial [[Membrane]] compared to biological ones.

Decreasing temps incorporate more unsaturated and shorter fats
Increasing temps incorporate more saturated and longer fats

[[Lipid]]s can go from one leaflet to another in [[Bilayer]]s but it is very slow, and requires a significant energy barrier to be passed through. 

The leaflets can also have different compositions between the inner and outer membranes.

There are 3 kinds of connections between a [[Membrane]].

[[Integral Membrane Protein]]
[[Peripheral Membrane Protein]]
[[Lipid-Linked Protein]]

Small nonpolar molecules can transfer through a membrane via diffusion, but larger non-polar molecules will not.

Active transport releases energy
Passive transport requires energy
The type of energy required is:
- ATP
- Ion Gradients
- Electron Transport

Porins and Ion Channels both enable passive transport specifically. 
Porins have 3 subunits
Ion Channels have 4 subunits
Ion channels are **VERY SELECTIVE** compared to Porins.

A transporter/carrier protein do not have a completely through pore in the protein. It binds to the molecule to transfer.

Uniport transfers 1 molecule
Symport transfers 2 molecules in the same direction
Antiport transfers 2 molecules in different directions

Primary active transporters use ATP as the energy source
Secondary active transporters use Ion Gradients as the energy source

[[Peripheral Membrane Protein]]s are easier to dissociate than [[Lipid-Linked Protein]]s

## Metabolism
Involves [[Metabolic Pathway]]s which all share similar fundamental chemical and thermodynamic principles.

[[Metabolic Pathway]]s are irreversible and need to be spontaneous, but the individual steps may be reversible.

[[Cofactor]]s are non-protein structures that are used by [[Enzyme]]s to do their work (FAD, NAD+)
FAD is a [[Prosthetic Group]]

NAD and NADP both take 1 H and are [[Cofactor]]s
FAD takes 2 H and is a [[Cofactor]] and a [[Prosthetic Group]]

[[Catabolism]] and [[Anabolism]] go hand in hand, first making smaller parts from larger, and second making larger parts from smaller

[[Catabolism]] creates NADH NADPH and FADH2.
NADPH can be used for [[Anabolism]]
NADH and FADH2 can be used for [[Electron Transport Chain]] to create more ATP.

[[Catabolism]] breaks down [[Nucleic Acid]]s [[Protein]]s [[Polysaccharide]]s, and [[Lipid]]s.
[[Nucleic Acid]]s are not exactly a useful source of fuel.

[[Carbohydrate]]s are stored as Glycogen in the liver and skeletal muscle.
[[Fatty Acid]]s are stored as [[Lipid]]s

The biochemical standard state:
- pH = 7
- Reactant and Product concentrations = 1M
- Temp = 25C / 298K
- Pressure = 1atm
- \[H2O] = 55M

$G'\degree$ states biochemical standard state

[[Metabolic Pathway]]s typically are regulated synchronously with constant intermediate results within.

In [[Catabolism]] [[metabolite]]s are undergo [[Oxidation]] while [[Cofactor]]s are undergo [[Reduction]], typically using NAD, FAD, NADP

In [[Anabolism]] the opposite occurs, and typically uses NADPH and NADH

In [[Catabolism]], the [[Reduction]] of [[Cofactor]]s is called Dehydrogenation. (FAD -> FADH2)

ATP is one of the most useful [[Nucleic Acid]]s and this is typically created by [[Catabolism]]

The [[Oxidation]] of NADH/FADH2 creates 95% of ATP.

[[Thioester Bond]]s are very high energy similar to [[Phosphoanhydride Bond]]s

Coupled reactions are common and typically are caused by a shared intermediate molecule and a total free energy < 0

Reversible reactions can still use [[Enzyme]]s

## Oxidative Phosphorylation
[[Oxidative Phosphorylation]] is a major process used to create ATP. There are 2 major parts.

[[Oxidation]] of NADH/FADH2 

[[Phosphorylation]] where ADP creates ATP with an inorganic phosphate and Hydrogen added.

This all occurs in the inner mitochondrial membrane.

ATP synthesis is separate from the [[Electron Transport Chain]].

Most of the electron transport is done via Primary active transports.

4 [[Integral Membrane Protein]]s and 2 mobile electron carriers are used in the [[Electron Transport Chain]].

[[Cytochrome]]s are [[hemoprotein]]s that carry out electron transport. [[Hemoglobin]] is not a [[Cytochrome]] because it transfers Oxygen instead of electrons.

FMNH2 is similar to FADH2 but without the AD.

The co-enzyme [[Ubiquinone]] (Q) exists around Complex 2 and is a co-substrate.

Electrons move from lower affinity/low reduction potential to higher affinity/higher reduction potential.

Complex 1 has a higher reduction potential than Complex 4

NADH in this process is resoponsible for 10 hydrogen protons.

The matrix side is the negative side (fewer protons)

The intermembrane side is the positive side (more protons)

The movement of electrons through the [[Electron Transport Chain]] relies on proton movement through [[Integral Membrane Protein]]s

After the entire ETC is gone through, the Hydrogen protons in the Intermembrane side come back into the matrix through [[ATP Synthase]].

3 protons at a time  to create ATP from ADP.

Hydrogen protons cause rotation of the [[ATP Synthase]] to create ATP from ADP and inorganic phosphate.

When interacting with the [[CAC]] via Complex 2, there is no proton transfer in Complex 2, and FADH2 is oxidized to FAD.

FADH2 in this process is responsible for 6 protons.

[[Succinate]] turns into [[Fumarate]] and provides 2 hydrogens to FAD into FADH2
FADH2 turns back to FAD after it provides the 2 hydrongs to Q.

2x [[Cytochrome]] is required for either process.

[[Oxidation]] creates the gradient while [[Phosphorylation]] removes it.

ADP, inorganic Phosphate, and Hydrogen Protons are brought from the intermembrane space to the matrix.

ATP is reversed.

The rate of electron transport and oxygen consumption are relative to the rate of [[ATP Synthase]] via the magnitude of the proton gradient.

P/O ratio is the amount of ATP syntheesized to the amount of Oxygen consumed.
Typically 2.5 for NADH (10 H+ made / 4 H+ used)
1.5 for FADH2 (6 H+ made / 4 H+ used)

Uncolupling Proteins allow Hydrogen Protons to come back into the membrane without ATP synthesis. **Oxygen is still consumed**

In brown fat, an uncoupling protein exists that allows generation of heat when Hydrogen Protons transfer

[[Lipid]]-soluble drugs are also effective in hydrogen transfer through the membrane.

## Glycolysis
[[Glucose]] has a central position for cell [[Metabolism]].

[[Glycolysis]] is a [[Metabolic Pathway]] with [[Catabolism]] functions, creating 2 [[Pyruvate]] from 1 [[Glucose]], and also forms NADH and ATP.

Carbon is most oxidized with more Oxygen bonds (CO2)

6 CO2 is created from glucose [[Oxidation]]

More saturated [[Fatty Acid]]s will create more ATP

[[Glycolysis]] occurs in the cytosol, and serves as the first step for complete [[Glucose]] oxidation.

Processes 6-10 (Stage 2)happen twice for each [[Glucose]].

Substrate-level [[Phosphorylation]] is phosphoryl transfer from a molecule to a nucleoside triphosphate

There are 3 main enzymes used in the process:
- Process 1 uses [[Hexokinase]]
- Process 3 uses [[Phosphofructose kinase 1]]
- Process 10 uses [[Pyruvate kinase]]

ATP is consumed at processes 1 and 3
ATP is made at processes 6 and 10
NADH is made at process 7

2, 5, and 8 are isomerization reactions

6 is an [[Oxidation]] reaction

## The PDC
[[PDC]] stands for [[Pyruvate]] dehydrogenase complex and is the catalyst for [[Pyruvate dehydrogenase]]

This is a process that links [[Glycolysis]] to [[CAC]]

It occurs inside the matrix in the mitochondria, instead of in the [[Membrane]].

[[Pyruvate]] needs to be transported across the [[Membrane]] using [[Pyruvate Translocase]] in order to undergo [[PDC]], typically has no issue crossing the Outer Mitochondrial [[Membrane]] though

A proton is also transferred alongside the [[Pyruvate]]

[[Coenzyme A]] (CoA) is essentially a very complicated structure with a [[Thiol]] bond at the end.

The product of the [[Pyruvate dehydrogenase]] reaction is Acetyl-CoA

[[Pyruvate]] is decarbolyzed, generated a [[Thioester Bond]], and oxidizes carbon and reduces NAD -> NADH.

This reaction is **irreversible** Compared to [[Glucose]] - [[Pyruvate]] ([[Glycolysis]]) and Acetyl-CoA - CO2 ([[CAC]])

When water is added to Acetyl-CoA [[Coenzyme A]] is provided, and is [[Exothermic]]

Using [[Coenzyme A]], [[Fatty Acid]]s can be activated by adding them together, releasing a water molecule and bonding CoA with a [[Thioester Bond]]

**Requires Aerobic Conditions in the matrix**
 
[[Pyruvate]] + [[Coenzyme A]] + NAD -> Acetyl-CoA + NADH + CO2

[[Phosphorylation]] will turn off the activity of [[PDC]]. Otherwise it will turn on.

[[Pyruvate]], [[Coenzyme A]], NAD, and ADP, and Ca2+ will inhibit the dephosphorylation, while water, Acetyl-CoA, and NADH encourage dephosphorylation

Oxygen concentration is inversely proportional to ATP concentration, and Oxygen Consumption Rate 

## The CAC
The entire cycle occurs in the matrix of the mitochondria.

The process is the [[Oxidation]] of the acetate in Acetyl-CoA into oxaloacetate (2C -> 2CO2)

The process is [[Amphibolic]]

Acetyl-CoA combined with oxaloacetate  to create citrate, and 2 carbons are lost. 2NADH is also made.

oxaloacetate is typically regenerated and creates another NADH FADH and GTP

An [[Energy Capture Step]] is a reaction that creates a high-energy molecule such as NADH FADH GTP.

alpha-Ketoglutarate is very similar to the [[PDC]], but it is not regulated, and Ca2+ is not completely used

the difference between synthase and synthetase is that a nucleotriophosphate is used.

Reaction 6 in the cycle is catalyzed by Complex 2 from the [[Electron Transport Chain]]

The order of reactions:
- Condensation
- Isomerization
- Oxidative Decarboxylation, 
- Thioester bond formation
- SLP
- Oxidation
- Hydration
- Oxidation

High ATP, NADH, or FADH2 will decrease the rate of the [[CAC]]

[[Amino Acid]]s can come in various forms into the [[CAC]] while [[Fatty Acid]]s can only come through Acetyl-CoA.

[[Anapleurotic]] reactions are used to handle [[CAC]] intermediates

The [[CAC]] provides precursors for biosynthesis, and is important for the generation of ATP in general.

From [[Glucose]] to the full cycle of [[CAC]], a total of 32 ATP is made (10 NADH, 2 FADH, 2 GTP) and 6 CO2 is produced

From one glucose in aerobic conditions
- [[Glycolysis]] generates 2 ATP + 2 NADH (7 ATP) + 2 Pyruvate
- 2[[PDC]] generates 2 NADH (5 ATP) + 2 Acetyl-CoA
- 2[[CAC]] generates 6 NADH (15 ATP) + 2 FADH2 (3 ATP) + 2 GTP (2 ATP)
- [[Lactate]] oxidized to [[Pyruvate]] generates 1 NADH

From one glucose in anaerobic conditions
- [[Glycolysis]] generates 2 ATP + 2 NADH (not used for ATP)
- [[PDC]] and [[CAC]] do not occur.

When the process is anaerobic, [[Pyruvate]] makes [[Lactate]] instead of Acetyl-CoA, and forms NAD+, which can be used in [[Glycolysis]] again.

When [[Lactate]] is used to run the [[CAC]], a total of 15 ATP is made.

Each NADH/FADH2 used in the [[Electron Transport Chain]] creates 1 water molecule.

[[Lactate]] -> [[Pyruvate]] forms 1 water (NADH) (UNDER AEROBIC CONDITIONS)
[[PDC]] forms 1 water (NADH)
[[CAC]] forms 4 water (3NADH + 1FADH2)
6 water is ultimately formed



Less ADP slows down the processes, while more ADP provides more incentive to run the processes to form ATP

[[Gluconeogensis]] causes the opposite of [[Glycolysis]], but neither [[Glycolysis]] itself is not reversible.

